src/App.tsx

```diff
@@ -38,24 +34,23 @@ export enum AlertLevel {
   NONE
 }
 
-interface L1ToL2MessageReaderWithNetwork extends IL1ToL2MessageReader {
+interface L1ToL2MessageReaderWithNetwork extends L1ToL2MessageReader {
   l2Network: L2Network;
 }
-
 const looksLikeCallToInboxethDeposit = async (
-  l1ToL2Message: IL1ToL2MessageReader
+  l1ToL2Message: L1ToL2MessageReader
 ): Promise<boolean> => {
-  const txData = await l1ToL2Message.getInputs();
-
+  const txData = await l1ToL2Message.messageData;
   return (
     txData.l2CallValue.isZero() &&
-    txData.maxGas.isZero() &&
-    txData.gasPriceBid.isZero() &&
-    txData.callDataLength.isZero() &&
-    txData.destinationAddress === txData.excessFeeRefundAddress &&
+    txData.gasLimit.isZero() &&
+    txData.maxFeePerGas.isZero() &&
+    ((txData.data.toString() === ethers.constants.HashZero) || (txData.data.toString() === "")) &&
```

Comment:
You mentioned you tested this to see the value, but I'm kinda surprised it's equal to HashZero, i thought it might just be the value '0x'.

Reply:
Yeah agreed! I've added this one as well! Thanks Chris!